### PR TITLE
glTF loading validation

### PR DESCRIPTION
**Pull request by AnastaZIuk**

### CODE REVIEW

File: src/nbl/asset/interchange/CGLTFLoader.cpp [hunk 1]
```diff
@@ -119,9 +108,17 @@ namespace nbl
 				_file->read(future, jsonBuffer->getPointer(), 0u, jsonBuffer->getSize());
 				future.get();
 			}
-			simdjson::dom::object tweets = parser.parse(reinterpret_cast<uint8_t*>(jsonBuffer->getPointer()), jsonBuffer->getSize());
-			simdjson::dom::element element;
 
+			simdjson::dom::object tweets;
+			auto error = parser.parse(reinterpret_cast<uint8_t*>(jsonBuffer->getPointer()), jsonBuffer->getSize()).get(tweets);
+#
```
Comment (devshgraphicsprogramming) [inline src/nbl/asset/interchange/CGLTFLoader.cpp:114]: typo?

Comment (AnastaZIuk) [reply src/nbl/asset/interchange/CGLTFLoader.cpp:114]: Where? There is no typo

Comment (devshgraphicsprogramming) [reply src/nbl/asset/interchange/CGLTFLoader.cpp:114]: the `#` randomly in the code?

Comment (AnastaZIuk) [reply src/nbl/asset/interchange/CGLTFLoader.cpp:114]: oh you're right, didn't notice it

**Pull request merged**